### PR TITLE
Add dockerfile and Github action

### PR DESCRIPTION
**Pull request by daxmc99**
This adds a Dockerfile and ~~a Github action so that this repository can update itself.~~

### CODE REVIEW

File: .github/workflows/update-docker-tags.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+      - uses: actions/checkout@v2.2
+      - name: test
+        # Debug
+        run: |
+          git status
+      - name: Update-tags
+        uses: ./.
+      - name: pull-request
+        uses: repo-sync/pull-request@v2
```
Comment (slimsag) [inline .github/workflows/update-docker-tags.yml:21]: that's neat! I didn't know this existed

**Comment by slimsag** [comment]
The Dockerfile being in this repo makes sense, and I can see having the GitHub action in e.g. the README as an example making sense, but I think it doesn't/shouldn't actually be an action that runs in my repository, right?

We probably should put that in the sourcegraph repository instead

**Comment by daxmc99** [comment]
That makes sense. I still think it needs some more testing though. I have some runs in my repo https://github.com/daxmc99/update-docker-tags/actions?query=workflow%3A.github%2Fworkflows%2Fupdate-docker-tags.yml but I'm having trouble determining if it working now.

**Comment by daxmc99** [comment]
I dropped the Github action from this commit so now it is just a Dockerfile additon

**Comment by slimsag** [comment]
Super sorry for the late review, will review ASAP.

**Comment by daxmc99** [comment]
Superceded by https://github.com/sourcegraph/update-docker-tags/

**Pull request closed without merging**